packages/dev/core/src/Materials/Textures/renderTargetTexture.ts

```diff
@@ -475,7 +475,7 @@ export class RenderTargetTexture extends Texture implements IRenderTargetTexture
             type = options.type ?? Constants.TEXTURETYPE_UNSIGNED_INT;
             isCube = !!options.isCube;
             samplingMode = options.samplingMode ?? Texture.TRILINEAR_SAMPLINGMODE;
-            generateDepthBuffer = !!options.generateDepthBuffer;
+            generateDepthBuffer = options.generateDepthBuffer ?? true;

```

Comment:
To be consistent, we should update `doNotChangeAspectRatio` also to default to `true`. Also, if you don't mind, it would be good to update the comments for the `RenderTargetTextureOptions` interface to indicate what the default is for each option. I should have done that to begin with. Thanks for pointing this out.

Reply:
Should `RenderTargetTextureOptions` be changed to extend [`RenderTargetCreationOptions`](https://github.com/BabylonJS/Babylon.js/blob/ae5d9a12d0a7731a2c61796272b82d1ddba3b8dd/packages/dev/core/src/Materials/Textures/textureCreationOptions.ts#L32) instead of duplicating a lot of those options?

Reply:
I didn't because they are not the same (though there is overlap) and also not intended for the same scope.